docs/en/docs/contributing.md

```diff
@@ -84,7 +84,17 @@ To check it worked, use:
 
 If it shows the `pip` binary at `env/bin/pip` then it worked. 🎉
 
+Make sure you have the latest pip version on your virtual environment to avoid errors on the next steps:
 
+<div class="termy">
+
+```console
+$ python -m pip install --upgrade pip
```

Comment:
FYI, this command helped me to solve errors when attempting to install `Flit` since I had and old `pip` version, .i.e.

```
$ python -m pip install --upgrade pip
Cache entry deserialization failed, entry ignored
Collecting pip
  Downloading https://files.pythonhosted.org/packages/a4/6d/6463d49a933f547439d6b5b98b46af8742cc03ae83543e4d7688c2420f8b/pip-21.3.1-py3-none-any.whl (1.7MB)
    100% |████████████████████████████████| 1.7MB 816kB/s 
Installing collected packages: pip
  Found existing installation: pip 9.0.1
    Uninstalling pip-9.0.1:
      Successfully uninstalled pip-9.0.1
Successfully installed pip-21.3.1
```